### DISCUSSION TITLE
500 error on login page in 6.1.15 (unable to locate file in Vite manifest)

### DISCUSSION THREAD

**Discussion by death2cupbots** [firefly-iii/firefly-iii#8877] [Firefly III] [answered]
When I open Firefly III on my laptop, I get a 500 error.

When I go to Firefly III on my phone (which is logged in) it works fine.

In order to update Firefly III, I'm running:
`wget https://github.com/firefly-iii/firefly-iii/releases/download/[currentverison]/FireflyIII-[currentversion].zip`

where currentversion is something like v6.1.15

I then unzip that, move stuff around, and then do:
`rm -rf  bootstrap/cache/*`
`php artisan clear`
`php artisan migrate --seed`
`php artisan firefly-iii:update-database`
`php artisan firefly-iii:laravel-passport-keys`
`php artisan cache:clear`

My PHP version is 8.3.3-1+ubuntu20.04.1+deb.sury.org+1
My Laravel version is 11.4.0
Is my Laravel out of date?

This is the error I get:
`Uncaught Exception: Unable to locate file in Vite manifest: resources/assets/v2/sass/app.scss. in /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Vite.php:781 Stack trace: #0 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Vite.php(299): Illuminate\Foundation\Vite->chunk() #1 /srv/firefly-iii/storage/framework/views/db94236f7f79405c2b540b1539192d4b.php(65): Illuminate\Foundation\Vite->__invoke() #2 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php(123): require('...') #3 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php(124): Illuminate\Filesystem\Filesystem::Illuminate\Filesystem\{closure}() #4 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(58): Illuminate\Filesystem\Filesystem->getRequire() #5 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(73): Illuminate\View\Engines\PhpEngine->evaluatePath() #6 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/View.php(208): Illuminate\View\Engines\CompilerEngine->get() #7 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/View.php(191): Illuminate\View\View->getContents() #8 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/View.php(160): Illuminate\View\View->renderContents() #9 /srv/firefly-iii/storage/framework/views/6ed1da0f6468d769b269b5f73600a743.php(75): Illuminate\View\View->render() #10 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php(123): require('...') #11 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php(124): Illuminate\Filesystem\Filesystem::Illuminate\Filesystem\{closure}() #12 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(58): Illuminate\Filesystem\Filesystem->getRequire() #13 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(73): Illuminate\View\Engines\PhpEngine->evaluatePath() #14 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/View.php(208): Illuminate\View\Engines\CompilerEngine->get() #15 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/View.php(191): Illuminate\View\View->getContents() #16 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/View.php(160): Illuminate\View\View->renderContents() #17 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #18 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Http/Response.php(35): Illuminate\Http\Response->setContent() #19 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Routing/ResponseFactory.php(58): Illuminate\Http\Response->__construct() #20 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Routing/ResponseFactory.php(88): Illuminate\Routing\ResponseFactory->make() #21 /srv/firefly-iii/app/Exceptions/Handler.php(181): Illuminate\Routing\ResponseFactory->view() #22 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(219): FireflyIII\Exceptions\Handler->render() #23 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(196): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse() #24 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(256): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException() #25 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}() #26 {main} Next Illuminate\View\ViewException: Unable to locate file in Vite manifest: resources/assets/v2/sass/app.scss. (View: /srv/firefly-iii/resources/views/layout/v2/error.blade.php) in /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Vite.php:781 Stack trace: #0 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(60): Illuminate\View\Engines\CompilerEngine->handleViewException() #1 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(73): Illuminate\View\Engines\PhpEngine->evaluatePath() #2 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/View.php(208): Illuminate\View\Engines\CompilerEngine->get() #3 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/View.php(191): Illuminate\View\View->getContents() #4 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/View.php(160): Illuminate\View\View->renderContents() #5 /srv/firefly-iii/storage/framework/views/6ed1da0f6468d769b269b5f73600a743.php(75): Illuminate\View\View->render() #6 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php(123): require('...') #7 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php(124): Illuminate\Filesystem\Filesystem::Illuminate\Filesystem\{closure}() #8 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(58): Illuminate\Filesystem\Filesystem->getRequire() #9 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(73): Illuminate\View\Engines\PhpEngine->evaluatePath() #10 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/View.php(208): Illuminate\View\Engines\CompilerEngine->get() #11 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/View.php(191): Illuminate\View\View->getContents() #12 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/View.php(160): Illuminate\View\View->renderContents() #13 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #14 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Http/Response.php(35): Illuminate\Http\Response->setContent() #15 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Routing/ResponseFactory.php(58): Illuminate\Http\Response->__construct() #16 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Routing/ResponseFactory.php(88): Illuminate\Routing\ResponseFactory->make() #17 /srv/firefly-iii/app/Exceptions/Handler.php(181): Illuminate\Routing\ResponseFactory->view() #18 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(219): FireflyIII\Exceptions\Handler->render() #19 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(196): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse() #20 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(256): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException() #21 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}() #22 {main} Next Illuminate\View\ViewException: Unable to locate file in Vite manifest: resources/assets/v2/sass/app.scss. (View: /srv/firefly-iii/resources/views/layout/v2/error.blade.php) (View: /srv/firefly-iii/resources/views/layout/v2/error.blade.php) in /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Vite.php:781 Stack trace: #0 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(60): Illuminate\View\Engines\CompilerEngine->handleViewException() #1 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(73): Illuminate\View\Engines\PhpEngine->evaluatePath() #2 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/View.php(208): Illuminate\View\Engines\CompilerEngine->get() #3 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/View.php(191): Illuminate\View\View->getContents() #4 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/View/View.php(160): Illuminate\View\View->renderContents() #5 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Http/Response.php(35): Illuminate\Http\Response->setContent() #7 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Routing/ResponseFactory.php(58): Illuminate\Http\Response->__construct() #8 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Routing/ResponseFactory.php(88): Illuminate\Routing\ResponseFactory->make() #9 /srv/firefly-iii/app/Exceptions/Handler.php(181): Illuminate\Routing\ResponseFactory->view() #10 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(219): FireflyIII\Exceptions\Handler->render() #11 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(196): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse() #12 /srv/firefly-iii/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(256): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException() #13 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}() #14 {main} thrown`

**Comment by JC5** [comment, marked as answer]
Run `php artisan view:clear`, that will probably clear it up.

**Reply by death2cupbots** [reply]
That fixed it. Thanks!
